### PR TITLE
Dacia spring charging

### PR DESCRIPTION
**Pull request by JLorenz86**
I added the function "setChargePause". This function is neccessary for stop and start charging the Dacia Spring.

### CODE REVIEW

File: src/ZEServices.ts [hunk 1]
```diff
@@ -562,6 +562,29 @@ export class ZEServices {
...
+    async setChargePause(pause: boolean, accountId: string, vin: string, country?: string) {
+        
+        let data: DataContainer<any> =
+        {
+            "data":
+            {
+                "type": "ChargePauseResume",
+                "attributes":
+                {
+                    "action": action ? "pause" : "resume"
```
Comment (konne) [inline src/ZEServices.ts:580]:
```suggestion
        const data: DataContainer<any> =
        {
            "data":
            {
                "type": "ChargePauseResume",
                "attributes":
                {
                    "action": pause ? "pause" : "resume"
```

Comment (konne) [reply src/ZEServices.ts:580]: @JLorenz86 please adapt the current code. It does not compile and I also made a small codesmell fix

**Review by konne — CHANGES_REQUESTED** [review]

**Comment by konne** [comment]
@JLorenz86 I now also enabled CI checks for you so you can see if the compile steps get green.

**Comment by JLorenz86** [comment]
Now, it should work.

**Comment by JLorenz86** [comment]
I have discovered another error in my version: The function for the Dacia spring requires a different base URL. I will add it this evening.

**Comment by JLorenz86** [comment]
I will fix the failures in my code and reopen an new pull request after that.

**Pull request closed without merging**